wire-ios/Wire-iOS/Resources/Base.lproj/Localizable.strings

```diff
@@ -290,8 +290,7 @@
 "conversation.input_bar.audio_message.keyboard.record_tip" = "Tap to record\nYou can  %@  it after that";
 "conversation.input_bar.audio_message.keyboard.filter_tip" = "Choose a filter above";
 
-"conversation.call.many_participants_confirmation.title" = "Start a call";
-"conversation.call.many_participants_confirmation.message" = "This will call %d people";
+"conversation.call.many_participants_confirmation.title" = "Are you sure you want to call %d people?";
```

Comment:
Just double checking that this copy came from the product / copy team and not from developers?

Reply:
Yep, that came directly from Astrid. She left a comment on the ticket.